README.md

```diff
@@ -6,67 +6,42 @@
+## Installation: Managed Karpenter (AKA Node Auto Provisioning) 
+The Node Auto Provisioning Preview, runs Karpenter as a managed addon similar to Managed Cluster Autoscaler. 
 
-Once done, you can delete all infra with `make az-rmrg` (it deletes the resource group), and can delete the codespace (though it will be automatically suspended when not used, and deleted after 30 days.)
+To get started, just go through the prerequisites of [installing the Preview CLI](https://learn.microsoft.com/en-gb/azure/aks/node-autoprovision?tabs=azure-cli#install-the-aks-preview-cli-extension), and [register the NodeAutoProvisioningPreview feature flag](https://learn.microsoft.com/en-gb/azure/aks/node-autoprovision?tabs=azure-cli#register-the-nodeautoprovisioningpreview-feature-flag).
 
-#### Developer notes
-- During step 1 you will observe `Running postCreateCommand...` which takes ~10+ minutes. You don't have to wait for it to finish to proceed to step 2.
-- The following errors can be ignored during step 2:
+### Enable node autoprovisioning 
+To enable node autoprovisioning, create a new cluster using the `az aks create` command and set --node-provisioning-mode to "Auto". You'll also need to use overlay networking and the cilium network policy for now.
 
+```bash 
+az aks create --name myFirstNap --resource-group napTest --node-provisioning-mode Auto --network-plugin azure --network-plugin-mode overlay --network-dataplane cilium
 ```
-ERRO[0007] gcloud binary not found
-...
-ERRO[0003] gcloud binary not found
-...
-ERRO[0187] walk.go:74: found symbolic link in path: /workspaces/karpenter/charts/karpenter/crds resolves to /workspaces/karpenter/pkg/apis/crds. Contents of linked file included and used  subtask=0 task=Render
-```
-- If you see platform architecture error during `skaffold debug`, adjust (or comment out) `--platform` argument.
-- If you are not able to set/hit breakpoints, it could be an issue with source paths mapping; see comments in debug launch configuration (`launch.json`)
-
-#### FAQs
 
-Q: I was able to trigger Karpenter to execute scaling up nodes as expected, using my own customized deployment of pods. However, scaling down was not handled automatically when I removed the deployment. The two new nodes created by Karpenter were left around. What is going on?
+[View Limitations of the node autoprovisioning preview here](https://learn.microsoft.com/en-gb/azure/aks/node-autoprovision?tabs=azure-cli#limitations)
```

Comment:
Should we maintain the list of limitations here instead?

Reply:
Better not to keep two sources of truth here I'd say. That link doesn't seem to capture the aks karpenter provider API version, but I think that's fine. If we need to add self-hosted specific limitations, then we could add a few bullet points here as needed. 

Might be worth calling out that that links to some information only relevant for our managed experience, but that the   `Limitations` section is relevant to self-hosted.

Reply:
Right now we don't have a self hosted experience besides makefile cmmands, so we should probably wait until we do